sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -376,14 +376,6 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putBoolean(KEY_SHARING, sharing) }
     }
 
-    internal fun getGooglePlayProjectNumber(context: Context): Long {
```

Comment:
nit: remove the `KEY_GOOGLE_PLAY_PROJECT_NUMBER` from above

Reply:
Done